uasc/config.go

```diff
@@ -114,6 +114,10 @@ type SessionConfig struct {
 	// The SignatureData type is defined in 7.32.
 	UserTokenSignature *ua.SignatureData
 
+	// SessionName is an optional name of the session.
+	// The default is a random value.
```

Comment:
`UnixNano` is not random

Reply:
No, it isn't. Clarified the documentation. Question is whether it has to be.